services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -280,9 +280,9 @@ public class VeniceControllerConfig extends VeniceControllerClusterConfig {
 
   private final int storeGraveyardCleanupSleepIntervalBetweenListFetchMinutes;
 
-  private final boolean systemStoreHealthCheckEnabled;
+  private final boolean systemStoreRepairServiceEnabled;
 
-  private final int systemStoreHealthCheckIntervalSeconds;
+  private final int systemStoreRepairCheckIntervalHours;
```

Comment:
I think `hour` as the config unit is too large, and we should continue to use `seconds`.
For example, if there is a problem with meta system store, repairing it after 6 hours will still trigger a GCN.
DaVinci is still using meta system store today.

Reply:
But repairing for once could take up to 1h if the push is stuck. 
If the previous run is not complete, we should wait until next run right? So I think in theory it should be at least in hour level...

Reply:
Were you referring to push job? 
The issue can happen to both DVC push system store and meta system store.

Here is an example:
1. User adds a new value schema.
2. User starts using this schema in. the nearline job.
3. DaVinci could receive a message encoded by the newly registered schema.
4. If meta system store is broken, DaVinci app won't be able to decode the payload.